indexing-service/src/test/java/io/druid/indexing/overlord/TestRemoteTaskRunnerConfig.java

```diff
@@ -33,7 +33,8 @@ public Period getTaskAssignmentTimeout()
   @Override
   public long getMaxZnodeBytes()
   {
-    return 1000;
+    // make sure this is large enough, otherwise RemoteTaskRunnerTest might fail unexpectedly
```

Comment:
I'm not sure how the failure happens, but is it clear that it's an issue with this config?  If not, is there a way to make that clear?